packages/widget_3d_tiles/src/index.js

```diff
@@ -7,6 +7,47 @@ const DEFAULT_OPTIONS = {
   position: 'top-right', // should be deprecated https://github.com/iTowns/itowns/issues/2005
 };
 
+/**
+ * Within a list of already existing layers, add a UI entry for a newly
+ * appended layer (together with a button for its removal).
+ *
+ * @param {itowns.View} view - The view to which the layer will be added
+ * @param {itowns.C3DTilesLayer} layer - layer to be added to dom element
+ * @param {HTMLDivElement} layersContainer - HTML division holding the listed layers
+ * @param {string} layerContainerClassName - Class name of the layer container
```

Comment:
layerContainer is optional the JSDOC syntax is:
`* @param {string} [layerContainerClassName] - Class name of the layer container`